python/paddle/fluid/imperative/layers.py

```diff
@@ -36,14 +36,31 @@ class Layer(core.Layer):
         dtype: data type for the variables in the layer.
     """
 
-    def __init__(self, name_scope, dtype=core.VarDesc.VarType.FP32):
+    def __init__(self,
+                 name_scope,
+                 dtype=core.VarDesc.VarType.FP32,
+                 act=None,
+                 use_cudnn=None,
+                 use_mkldnn=None,
+                 param_attr=None,
```

Comment:
this shouldn't live in Layer?

Reply:
we need to pass this args to allow user set different layer_helper instance from base Layer